build_runner/test/generate/watch_test.dart

```diff
@@ -816,6 +848,9 @@ Future<BuildState> startWatch(List<BuilderApplication> builders,
       logLevel: logLevel,
       onLog: onLog,
       skipBuildScriptCheck: true);
+  // Some tests need access to `reader` so we expose it through an expando.
```

Comment:
This feels a little strange to me.

Reply:
Ya it is but its just a test and its a low boilerplate/simple way to get ahold of this reader from the test.

I could also create a whole new class which has the reader and BuildState object and returning that (and that is what I would do if this was public code that needed this), but that just seemed overkill for this use case.